src/coreclr/vm/jitinterface.cpp

```diff
@@ -11608,45 +11608,19 @@ void CEEJitInfo::allocUnwindInfo (
 
     memcpy(pUnwindInfoRW, pUnwindBlock, unwindSize);
 
-#if defined(TARGET_X86)
-
-    // Do NOTHING
-
-#elif defined(TARGET_AMD64)
-
+#if defined(TARGET_64BIT) && defined(TARGET_WINDOWS)
```

Comment:
```suggestion
#if !defined(TARGET_X86) && defined(TARGET_WINDOWS)
```
This would be more logical ifdef for this one.

Also, `reservePersonalityRoutineSpace` above can be wrapped with the same ifdef now that we are not filling in the personality routine on non-Windows.